appendices/VK_EXT_acquire_drm_display.txt

```diff
@@ -0,0 +1,29 @@
+// Copyright 2018-2021 The Khronos Group, Inc.
+//
+// SPDX-License-Identifier: CC-BY-4.0
+
+*Last Modified Date*::
+    2021-06-08
+*IP Status*::
+    No known IP claims.
+*Contributors*::
+  - Simon Zeni, Status Holdings, Ltd.
+
+=== Description
+
+This extension allows an application to take exclusive control of a display
+using the Direct Rendering Manager (DRM) interface. When acquired, the display
+will be under full control of the application until the display is either
+released, the provided DRM fd is closed, or the physical connection to the
+display is lost.
+
+include::{generated}/meta/{refprefix}VK_EXT_acquire_drm_display.txt[]
```

Comment:
This include should go at the start of the appendix, followed by '=== Other Extension Metadata' and then the '*Last Modified Date*' etc.

Reply:
Then at this point (end of Description), add the generated interfaces include:

> include::{generated}/interfaces/VK_EXT_acquire_drm_display.txt[]